src/CUDAdrv.jl

```diff
@@ -3,21 +3,24 @@ __precompile__()
 module CUDAdrv
 
 using Compat
-import Compat.String
+using Compat.String
```

Comment:
So, I think in this case it doesn't actually matter too much since we already do `using Compat` so all the names are available anyways. The `using Compat.String` makes `String` available but not extendable, so it's safer in case we accidentally try to add a method to `String` (pedantic and unlikely, but I believe this is the distinction).

If on the other hand, we did not have `using Compat` and just wanted to use Compat's String method then `import Compat.String` is the correct/only way to bring into scope just that binding.

Please correct me if I'm wrong here.